worker-core/pom.xml

```diff
@@ -21,14 +21,14 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
-    <groupId>com.github.workerframework</groupId>
+    <groupId>com.github.workerframework.worker</groupId>
```

Comment:
This extra `.worker` is bothersome.  Put these in a spreadsheet and let's go through them.

Reply:
Ok let's review them today.